### DISCUSSION TITLE
quick question

### DISCUSSION THREAD

**Discussion by doublezeroseven** [MobiDAX/opendax#5] [General]
hi i have a quick question,

im trying to download this file:

https://docker.pkg.github.com/v2/mobidax/opendax/baseapp/manifests/2.3.2

using "docker" 🤔

but it's giving me this error:

{"errors":[{"code":"UNAUTHORIZED","message":"Your request could not be authenticated by the GitHub Packages service. Please ensure your access token is valid and has the appropriate scopes configured."}]}

😕

**Comment by bandexcoin-project** [comment]
https://youtu.be/qLjl2csyVPw?t=1000
